FlowCrypt/Functionality/Services/GlobalRouter.swift

```diff
@@ -123,6 +123,7 @@ extension GlobalRouter: GlobalRouterType {
         proceed(with: session)
     }
 
+    @MainActor
```

Comment:
We could go to this method only from `proceed` and it is already marked with `@MainActor`. Are you sure we need this one?

Reply:
I do have compiler error if this method is not marked as `@MainActor`

```
Call to main actor-isolated initializer 'init(error:encryptedStorage:router:)' in a synchronous nonisolated context
```